SiteOps/ApproveRules.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
+import sys
```

Comment:
Please at least make this as compatible as possible with python3 by adding `from __future__ import absolute_import, division, print_function` and adjusting the print functions.

Reply:
Added from __future__ import absolute_import, division, print_function and adjusted print functions.